packages/core/README.md

```diff
@@ -540,8 +540,8 @@ Unless otherwise stated, all validators require non-null and non-undefined value
 | nullOrUndefined         |                                                                  | Requires null or undefined value.                                                                                                          |
 | notEmpty                |                                                                  | Requires non-null, non-undefined and not empty. Works with anything having numeric `length` property, e.g. string or array.                |
 | notBlank                |                                                                  | Requires non-null, non-undefined and not blank (whitespace only string) values.                                                            |
-| uuid                    | version?: number                                                 | Uses `uuid-validate` package to validate input.                                                                                            |
-| pattern                 |  pattern: string \| RegExp, flags?: string                       | Tests input against the pattern.                                                                                                           |
+| uuid                    | version?: number                                                 | Uses `uuid` package to validate input.                                                                                                     |
```

Comment:
Trim trailing spaces setting could be missing 🤔

Reply:
It's table formatting, making the columns in all the rows same size. Row ends with | :)